todo.md

```diff
@@ -0,0 +1,4 @@
+- add `.fill` to fill paths
+- add `.arc`, `.arcTo`
+- add `fillText`
+- think up more elegant way to call `.frame()`
```

Comment:
I just don't like that `process.stdout.write(canvas._canvas.frame())`, - it puts user into details of drawille-canvas implementation. There should be some simple method, like `.frame()` in drawille, I hoped there might be something closer to user's mind, i.e. already existing pattern of use, like canvas API `context.getImageData()`, streams `canvas.pipe(writable)`, or `canvas.toString()` maybe?